lib/fpm/package/freebsd.rb

```diff
@@ -128,8 +128,17 @@ def add_path(tar, tar_path, path)
     end
   end # def add_path
 
+  def to_s_extension; "txz"; end
+
+  def to_s_fullversion()
+    # iteration (PORTREVISION on FreeBSD) shall be appended only(?) if non-zero.
+    # https://www.freebsd.org/doc/en/books/porters-handbook/makefile-naming.html
+    return "#{to_s_version}_#{iteration}" if iteration and (iteration.to_i > 0)
```

Comment:
Or replace ``to_s_version`` with just ``version`` (to avoid interdependencies between ``to_s_*`` methods).